javalib/src/main/scala/java/net/Inet6Address.scala

```diff
@@ -8,11 +8,11 @@ import scalanative.unsigned._
 import scala.scalanative.posix.net.`if`._
 import scala.scalanative.posix.stddef
 
-final class Inet6Address private (
-    val ipAddress: Array[Byte],
+final class Inet6Address(
+    ipAddress: Array[Byte],
     host: String,
-    scopeId: Int,
-    val zoneIdent: String
+    private val scopeId: Int,
```

Comment:
If the scopeId is not referenced from other instances, and it seems it is not because it compiled previously, then we can keep it in the original form `copeId: Int`. It would be treated by the compiler similarly to  Scala 2 `private[this]` 
```scala 
scala> class Foo(x: Int){def show(other: Foo) = println(other.x) }
-- [E173] Reference Error: -----------------------------------------------------
1 |class Foo(x: Int){def show(other: Foo) = println(other.x) }
  |                                                 ^^^^^^^
  |   value x cannot be accessed as a member of (other : Foo) from class Foo.
1 error found
                   
```